package.json

```diff
@@ -35,6 +35,7 @@
     "eslint-plugin-react": "^7.4.0",
     "express": "^4.15.5",
     "html-webpack-plugin": "^2.30.1",
+    "image-webpack-loader": "^3.4.2",
```

Comment:
you dont use this package, please remove it.
npm uninstall image-webpack-loader